### PR TITLE
[DX-1495] improve content

### PR DESCRIPTION
**Pull request by letzya**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

[preview](https://deploy-preview-5019--tyk-docs.netlify.app/docs/nightly/apim)

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

___

### **PR Type**
documentation

___

### **Description**
- Improved the wording and structure of the API Management deployment options documentation for better clarity and flow.
- Updated contact and trial information to be more concise and user-friendly.
- Enhanced the self-managed licensing include file for consistency in button formatting and improved clarity.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>apim.md</strong><dd><code>Refactor and enhance API Management deployment options documentation</code></dd></summary>
<hr>

tyk-docs/content/apim.md

<li>Improved wording for clarity and conciseness.<br> <li> Reorganized sections for better flow.<br> <li> Updated contact and trial information.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5019/files#diff-955dad1c340a1116b954795e70d8170dd79f5a0ea27b163342c5592db831b580">+13/-16</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>self-managed-licensing-include.md</strong><dd><code>Update self-managed licensing include for consistency and clarity</code></dd></summary>
<hr>

tyk-docs/content/shared/self-managed-licensing-include.md

<li>Updated button formatting for consistency.<br> <li> Improved wording for clarity.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5019/files#diff-b46c343a3cf502455a2e6f2507461b16ef81846d3ef643d79786035d023f46e1">+3/-5</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: tyk-docs/content/apim.md [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
-menu: "main"
 linkTitle: API Management
 tags: ["Tyk API Management", "Licencing", "Open Source", "Self-Managed", "Tyk Cloud", "API Gateway"]
 description: "How to decide on which Tyk deployment option is best for you"
 aliases:
     - /getting-started/deployment-options/
 ---
 
-Tyk API Management deployment options are comprised of the various [open and closed source]({{< ref "tyk-stack" >}}) components.
+Tyk API Platform offers various deployment options, consisting of both [open and closed source]({{< ref "tyk-stack" >}})
```
Comment (letzya) [inline tyk-docs/content/apim.md]:
```suggestion
Tyk API Platform offers various deployment options, consisting of both [open source and proprietary]({{< ref "tyk-stack" >}})
```

File: tyk-docs/content/apim.md [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
 
-Tyk API Management deployment options are comprised of the various [open and closed source]({{< ref "tyk-stack" >}}) components.
+Tyk API Platform offers various deployment options, consisting of both [open source and proprietary]({{< ref "tyk-stack" >}})
+components.
 
-Which one is right for your organisation depends on your requirements and preferences.  Please contact us for help:
-
-{{< button_left href="https://tyk.io/contact/" color="green" content="Contact us" >}}
+Choosing the right one for your organization depends on your specific requirements and preferences.
+</br>Don’t hesitate to contact us for assistance {{< button_left href="https://tyk.io/contact/" color="green" content="Contact us" >}}
```
Comment (dcs3spp) [inline tyk-docs/content/apim.md:15]: Render contact us button a new line

```suggestion

</br>

Don’t hesitate to contact us for assistance {{< button_left href="https://tyk.io/contact/" color="green" content="Contact us" >}}
```

Comment (letzya) [reply tyk-docs/content/apim.md:15]: It's on purpose in the same line, to connect it to the text

File: tyk-docs/content/shared/self-managed-licensing-include.md [hunk 3]
```diff
@@ -6,9 +6,7 @@ Tyk Self-Managed is the easiest way to install Tyk Full Lifecycle API Management
...
+{{< button href="https://tyk.io/sign-up/" color="green" content="Free trial" >}}
 
-{{< button_left href="https://tyk.io/sign-up/" color="green" content="Free trial" >}}
-
-For longer duration trials, or to request trials of the other proprietary software please contact the Tyk Team and tell us about your plans:
-
-{{< button_left href="https://tyk.io/contact/" color="green" content="Contact us" >}}
\ No newline at end of file
+For longer duration trials, or to request trials of the other proprietary software please contact the Tyk Team and tell
+us about your plans: {{< button href="https://tyk.io/contact/" color="green" content="Contact us" >}}
```
Comment (letzya) [inline tyk-docs/content/shared/self-managed-licensing-include.md]:
```suggestion
us about your plans: {{< button_left href="https://tyk.io/contact/" color="green" content="Contact us" >}}
```

File: tyk-docs/content/shared/self-managed-licensing-include.md [hunk 4]
```diff
@@ -6,9 +6,7 @@ Tyk Self-Managed is the easiest way to install Tyk Full Lifecycle API Management
 See [licensing and deployment models]({{< ref "tyk-on-premises/licensing" >}}) to learn about the different deployment options for Self-Managed.
 
 Register here to get a 14-day temporary licenses for the Tyk Dashboard & Developer Portal:
+{{< button href="https://tyk.io/sign-up/" color="green" content="Free trial" >}}
```
Comment (letzya) [inline tyk-docs/content/shared/self-managed-licensing-include.md]:
```suggestion
{{< button_left href="https://tyk.io/sign-up/" color="green" content="Free trial" >}}
```

**Pull request merged**